app/assets/locales/android_translatable_strings.txt

```diff
@@ -116,11 +117,16 @@ sync.fail.bad.data=Server provided improperly formatted data, please try again o
 sync.fail.bad.local=Your information was fetched, but a problem occurred with the log in. Please try again.
 sync.fail.bad.network=Couldn't contact server. Please make sure an internet connection is available or try again later.
 sync.fail.server.error=The server encountered an error processing your data. Please try again later. If the problem persists, contact CommCare Support.
-sync.fail.rate.limited.server.error=Our servers are unavailable at this time. Please try again later.
+sync.fail.rate.limited.server.error=Form submission rate limit reached. Your forms are being saved locally and will be submitted to the server later.

```

Comment:
This string get used by data pull not form submission, so if user is coming here that means form submission has been successful already.

Reply:
Ohh, I missed that. The same string is also used to show rate limit error in form upload. It's better to separate the two strings.

Reply:
Ahh, I misused this key in the first place.